conf/vmware.yaml.template

```diff
@@ -0,0 +1,21 @@
+VMWARE:
+    AUTH:
+        VCENTER: <VCENTER>
+        USERNAME: <USERNAME>
+        PASSWORD: <PASSWORD>!
+    CRITERIA:
+        VM:
+            # The VM to be deleted with prepend string, e.g VM name that starts with 'test'
+            DELETE_VM: 'test'
+            # Number of minutes the deletable VM should be allowed to live, e.g 120 minutes = 2 Hours
+            SLA_MINUTES: 120
+        DISC:
+            UNASSIGNED: True
+        NIC:
+            UNASSIGNED: True
```

Comment:
I think you should remove this as we dont support this yet!

Reply:
IIRC, I had keep it for some validations in cloudwash CLI as this are part of common options, anyway we could leverage this when add support for it later